c36730805.lua

```diff
@@ -1,4 +1,5 @@
 --スピードリフト
+--Fixed by Lee
```

Comment:
git has `blame` feature which can see who changed each line so we don't need to add author info to the code